developer_manual/prologue/compatibility_app_ecosystem.rst

```diff
@@ -0,0 +1,31 @@
+====================================
+Compatibility with the app ecosystem
+====================================
+
+The Nextcloud app ecosystem consists of hundreds of apps and app maintainers.
+
+The development process may require changes that affect apps in this ecosystem. These changes may either be a change in standards or best practises, yet in more extreme cases the changes may also be breaking apps.
+
+While developers are expected to aim for limiting the number of changes that break apps, this cannot always be avoided or is not always reasonable to do to maintain an up-to-date technology stack.
+
+Informing app developers about any change that affect them is therefore key to a thriving ecosystem. This page describes the three procedures to document changes that are relevant to app developers.
+
+
+Documentation procedures of changes that affect app developers
+--------------------------------------------------------------
+There are three procedures in place to facilitate communication to app developers about changes that affect them:
+
+
+1. The definition of done of a pull request includes documentation.
+This means the development of your pull request cannot be considered done if you introduced changes that affect app developers that you did not document.
+
+2. A change that affects app developers, has to be reported and documented by the author of the pull request in the `Changes for developers and administrators issue <https://github.com/nextcloud/server/issues/34692>`_.
```

Comment:
The link is for 26 so it might be good to add updating that to the list of things to do during the version bump

Reply:
yes! I'm not sure which list it is or where to find it. any chance you can add that to this list?

Reply:
Ah was merged already, so it's in https://github.com/nextcloud/documentation/pull/9323

Reply:
thanks! ❤️